### PR TITLE
Bug fix and GPhL release version update

### PR DESCRIPTION
**Pull request by rhfogh**
Simple bug fix (missing import statement in recent commit).

An editor like Pycharm that highlights errors in red is nice to have here.

### CODE REVIEW

**Comment by rhfogh** [comment]
Added some real changes - upgraded handling of crystal symmetry for GPhL workflow automation

File: mxcubecore/HardwareObjects/Gphl/GphlMessages.py [hunk 1]
```diff
@@ -349,7 +349,7 @@ def __init__(
 
     @property
     def priorCrystalClasses(self):
```
Comment (marcus-oscarsson) [inline mxcubecore/HardwareObjects/Gphl/GphlMessages.py:351]: How about a nice type hint ;)

File: mxcubecore/HardwareObjects/Gphl/GphlWorkflow.py [hunk 2]
```diff
@@ -2083,17 +2093,41 @@ def select_lattice(self, payload, correlation_id):
...
+                )
+            params["space_group"] = space_group
+            params["crystal_classes"] = crystal_classes
+
+        else:
             params = self.query_pre_strategy_params(choose_lattice)
             if params is StopIteration:
                 return StopIteration
             indexing_solution = params["indexing_solution"]
+        #
```
Comment (marcus-oscarsson) [inline mxcubecore/HardwareObjects/Gphl/GphlWorkflow.py]: Extra hash :)

**Comment by marcus-oscarsson** [comment]
Thanks, looks good to me. Just a typo to fix perhaps.

File: mxcubecore/HardwareObjects/Gphl/GphlMessages.py (hunk 1 above)
Comment (rhfogh) [reply mxcubecore/HardwareObjects/Gphl/GphlMessages.py:351]: The entire file is minus type hints and doc strings, as are a couple of other GPhL files. Fixing it is on my TODO list (as indeed it should be), but I would rather do them all at once and *then* keep them up to date than add them piecemeal.

**Pull request merged**